guidelines.md

```diff
@@ -62,6 +63,18 @@ In case of any error, the returned JSON object describes the error and has the f
 
 Some errors may also contain additional information relevant to the error on top of these two properties. That depends on the operation and is specifically described in the operation documentation.
 
+### Request limits
+
+#### Demo environment
+* 2000 requests per `Client Token` within 15 minutes
+* 1000 requests per `Access Token` within 15 minutes
+* 500 requests per endpoint per `Access Token` within 15 minutes
+
+#### Production environment
+* 3600 requests per `Client Token` within 15 minutes
+* 1600 requests per `Access Token` within 15 minutes
+* 800 requests per endpoint per `Access Token` within 15 minutes
```

Comment:
All those references should not  have space between Access and Token to make it consistent with the rest. `AccessToken` and `ClientToken`

Reply:
fixed